packages/components/src/components/helpers/Input/Input.module.css

```diff
@@ -10,7 +10,7 @@
   position: relative;
 }
 
-.input {
+:where(.input) {
```

Comment:
Nice med mere `:where` 🤩 Tror det blir diggere for konsumenter som ønsker å overstyre styling også 😄